src/app/inventory/store/sockets.ts

```diff
@@ -416,12 +429,17 @@ function buildPlug(
       ).join('\n')
     : '';
 
+  const cannotCurrentlyRoll = Boolean(
+    plugSet?.plugs.find((p) => p.plugDef.hash === plugDef.hash)?.cannotCurrentlyRoll
+  );
```

Comment:
This condition should probably be inverted -- some weapons have two copies of a perk, one of which can roll and one of which can't (see #7272). So this should not check if there is an entry that cannot currently roll, it should instead check if there is no entry that can currently roll.

Otherwise I'm getting mismatches between items and Armory, this marks my Adept Hotheads as no longer rolling Tracking Module while Armory considers this perk totally fine.